gameplay/src/Vector2.h

```diff
@@ -413,6 +421,15 @@ class Vector2
     inline bool operator<(const Vector2& v) const;
 
     /**
+    * Determines if this vector is greater than the given vector.
+    *
+    * @param v The vector to compare against.
+    *
+    * @return True if this vector is greater than the given vector, false otherwise.
+    */
+    inline bool operator>(const Vector2& v) const;
```

Comment:
Do you have a need for this method? I think the only reason we implemented `operator<` is so that the vector could be used as a key in a map. The less than and greater than operators typically aren't useful in a vector. A lot of math APIs don't bother including it. (Like [XNA](https://msdn.microsoft.com/en-us/library/Microsoft.Xna.Framework.Vector2_methods.aspx) and [glm](http://glm.g-truc.net/0.9.7/api/a00134_source.html))